tst/CTA.WebForms2Blazor.Tests/FileConverterTests.cs

```diff
@@ -20,21 +20,23 @@ class FileConverterTests
         //These are relative paths
         private string _testProjectPath;
         private string _testCodeFilePath;
-        private string _testConfigFilePath;
+        private string _testWebConfigFilePath;
         private string _testStaticFilePath;
         private string _testViewFilePath;
         private string _testProjectFilePath;
+        private string _testAreaFullPath;
 
         [OneTimeSetUp]
         public void OneTimeSetup()
         {
             var workingDirectory = Environment.CurrentDirectory;
             _testProjectPath = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
             _testCodeFilePath = Path.Combine(TestFilesDirectoryPath, "TestClassFile.cs");
-            _testConfigFilePath = Path.Combine(TestFilesDirectoryPath, "SampleConfigFile.config");
+            _testWebConfigFilePath = Path.Combine(TestFilesDirectoryPath, "web.config");
             _testStaticFilePath = Path.Combine(TestFilesDirectoryPath, "SampleStaticFile.png");
             _testViewFilePath = Path.Combine(TestFilesDirectoryPath, "SampleViewFile.aspx");
             _testProjectFilePath = Path.Combine(TestFilesDirectoryPath, "SampleProjectFile.csproj");
+            _testAreaFullPath = Path.Combine(_testProjectPath, TestFilesDirectoryPath);
         }
 
         [SetUp]
```

Comment:
Delete this `[SetUp]` function if not being used

Reply:
Added a comment that it might be used later for future file converter tests